.github/fix-docs-version.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+#
+# This script takes care of setting proper version in docs.
+#
+set -eo pipefail
+
+VERSION=$1
+
+update_file_versions() {
+  local VERSION="$1"
+  local FILE="$2"
+  sed -i "s/<version>.*<\/version>/<version>${VERSION}<\/version>/g" README.md
```

Comment:
Maybe we should use `${VERSION}` instead of directly `README.md`. Also, this might be useful to do the replacement in future versions of documentation, to apply the replacement on different files.